docs/excel/excel-add-ins-tables.md

```diff
@@ -353,6 +353,35 @@ Excel.run(function (context) {
 }).catch(errorHandlerFunction);
 ```
 
+### AutoFilter
+
+> [!NOTE]
+> `Table.autoFilter` is currently available only in public preview. [!INCLUDE [Information about using preview APIs](../includes/using-excel-preview-apis.md)]
+
+As of the `ExcelApi 1.9` requirement set, add-ins have access to the table's [AutoFilter](/javascript/api/excel/excel.autofilter). An AutoFilter object is the entire filter structure of a table or range. All the aforementioned filter operations are compatible with the auto-filter. The single access point does make it easier to access and manage multiple filters.
```

Comment:
I'm confused by the fact that the note indicates `AutoFilter` is only available in Preview (and in fact, that *is* where the reference article lives in the Ref docs) -- but this sentence says it's part of Requirement Set 1.9.  Seems like it would be one or the other, right?  i.e., it's either in Preview OR it's in a requirement set -- can't be both.  (FWIW, the Excel requirement sets doesn't currently list requirement set 1.9...leading me to believe that 1.9 hasn't been released yet. If that's the case, we shouldn't be referencing 1.9 anywhere in the docs yet.)